.circleci/config.yml

```diff
@@ -9,3 +9,27 @@ workflows:
       - elixir/build_test
       - elixir/test
       - elixir/lint
+
+  publish:
+    jobs:
+      - elixir/build_test:
+          filters: &filters-publish  # this yaml anchor allows for reusing this filter set elsewhere
+            branches:
+              ignore: /.*/
```

Comment:
IIUC this causes `build_test` to be NOT executed for branches, which is not OK. Only the `hex_publish` task should be limited to run for tags only. So, do we need that filter at all?
```suggestion
```

Reply:
From what I understand, it will be executed for branches in the build workflow. If the filter wasn't specified here (in the publish workflow), it would run twice for all branch pushes. I can also confirm that from testing this earlier.

Reply:
OK, I get it now, but why is this a separate workflow? Wouldn't it be enough to add `hex_publish` as the last job in the `build` workflow (which should be then renamed) and only that job would have a filter rule disabling building on branches